postman_collection.json

```diff
@@ -1,6 +1,6 @@
 {
 	"info": {
-		"_postman_id": "afb5a3d7-203e-47d9-9e4d-44e8b4077d88",
+		"_postman_id": "5287ec83-213c-4342-b33d-86aee8c30717",
```

Comment:
Please submit this change to postman_collection.json as a separate PR.

Reply:
This is due to the feature-notification branch on Savvato repo being 2 commits behind. TRIB-87/88 is the branch of the latest develop.